client/src/js/base/__tests__/InputSave.test.js

```diff
@@ -4,9 +4,15 @@ import { Flex, FlexItem, Button } from "../index";
 
 describe("<InputSave />", () => {
     let props;
-    let wrapper;
-
-    it("renders correctly", () => {
+    let state;
+    const blur = jest.fn();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['blur' is assigned a value but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4770288)